### PR TITLE
meta-raspberrypi: update to latest on rocko

### PR DESCRIPTION
**Pull request by mirzak**
Fixes issues found during rocko testing, https://at.projects.genivi.org/wiki/display/GDP/GDP+Testing

### CODE REVIEW

**Comment by mirzak** [comment]
Hidden dependency in "rvi" on `install` applet. Exposed when `coreutils` where removed from target once we got rid of the -dev packages.

**Comment by mirzak** [comment]
Submitted a pull-request to rvi_core as well, https://github.com/GENIVI/rvi_core/pull/117

File: meta-genivi-dev/meta-rvi/recipes-connectivity/rvi/files/0001-rvi_ctl.template-get-rid-of-dependecy-on-install.patch [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+diff --git a/scripts/rvi_ctl.template b/scripts/rvi_ctl.template
+index 82ae11b..d93fe2a 100644
+--- a/scripts/rvi_ctl.template
++++ b/scripts/rvi_ctl.template
+@@ -104,7 +104,7 @@ then
+ 	else
+ 	    RUNDIR=${RVI_RUNDIR}
+ 	fi
+-	install -d --mode=0755 ${RUNDIR}
++	mkdir -p ${RUNDIR}
```
Comment (gunnarx) [inline meta-genivi-dev/meta-rvi/recipes-connectivity/rvi/files/0001-rvi_ctl.template-get-rid-of-dependecy-on-install.patch]: Does it need a  chmod 755?  (depends on whatever umask is active I guess - it's probably OK)

Comment (mirzak) [reply meta-genivi-dev/meta-rvi/recipes-connectivity/rvi/files/0001-rvi_ctl.template-get-rid-of-dependecy-on-install.patch]: Updated with `mkdir -m 755`

**Pull request merged**